Assets/MRTK/Providers/XRSDK/XRSDKDeviceManager.cs

```diff
@@ -125,6 +125,17 @@ public override void Update()
             }
         }
 
+        /// <inheritdoc/>
+        public override void Disable()
+        {
+            var controllersCopy = ActiveControllers.ToReadOnlyCollection();
+            foreach (var controller in controllersCopy)
+            {
+                CoreServices.InputSystem?.RaiseSourceLost(controller.Value.InputSource, controller.Value);
```

Comment:
Nit: It might be helpful to helper-ify these two lines since they are both critical in running during any remove operation (see lines 118-119 as well).

The risk is without in, someone could forget to have one of the lines (or if there's another change later that adds another operation here that's required, peopel might not update all of the occurneces)